app/views/content/non-uk-teachers/why-train-to-teach-in-england-video-transcript.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Why train to teach in England video transcript
+description: Full transcript for the 'Why train to teach in England' video.
+
+---
+
+## Teach in England
+
+**Christobel Gondwe, maths teacher:**
+
+If you want a job that you will love and you will feel wanted, and you will be looked after in essence, and you want a job where you can develop and progress, I’d definitely say to consider teaching in the UK. 
+
+## Is it hard to get on to a teacher training course in England? 
+
+**Nellie D’Costa, trainee science teacher:** 
+
+I assumed that it would be hard because I come from India. But then it was quite straightforward and easy. 
+
+**Giorgio Tatananni, trainee modern foreign languages teacher:**
+
+Getting a training place here was so much easier than I thought. 
+
+## How about sorting your visa and documents? 
+
+**Christobel Gondwe:** 
+
+Sorting out my visa was a lot easier than expected. The information on the UK Border Agency’s website was very comprehensive, and my school and training provider were very knowledgeable, so I had people holding my hand from start to finish. It was very easy to do. 
+
+## What support did you get? 
+
+**Nellie D’Costa:** 
+
+The support that I received when I applied for my training course was tremendous. I was supported at every step, right from the very beginning. Whenever I had a doubt, I would just call them up and they were more than happy to help me. 
+
+**Giorgio Tatananni:** 
+
+They are very thorough in your preparation. They never overload you with information. 
+
+**Christobel Gondwe:** 
+
+I felt that I had a lot of support. The application process was very straightforward. The process from start to finish was fantastic. 
+
+## Did the training meet your expectations? 
+
+**Giorgio Tatananni:** 
+
+The teacher training did not meet my expectation, it exceeded my expectation, simply because of all the support you get and all the help you have from your advisers and mentors. 
+
+**Nellie D’Costa:** 
+
+The training did meet my expectations and it was much more than I expected. It was really good because there are so many things that I didn’t know before, even though I was teaching, that I’ve learned here. So for me, it has been a fruitful experience. 
+
+## Anything else you’d like to tell us? 
+
+**Nellie D’Costa:** 
+
+The value of gaining a transferable qualification through training to teach in England is that you can go anywhere in the world and you can teach, and you can go in that part of the world and you can make a difference, not just in the lives of the children but also in the people around you.
+
+## Learn more about English teacher training
+
+Learn more about [training to teach in England as a non-UK citizen](/non-uk-teachers/train-to-teach-in-england-as-an-international-student)
```

Comment:
```suggestion
Learn more about [training to teach in England as a non-UK citizen](/non-uk-teachers/train-to-teach-in-england-as-an-international-student).
```
This is different from the video intentionally as the CTA is to go to the GIT website but anyone reading the transcript is already on it

However, we do need to add a full-stop after the link - our house style